### PR TITLE
Feature/link up payload and send issue

### PR DESCRIPTION
**Pull request by ficklephil**
- POST'ing successfully from Node to Api.
- organisation name, superUser, and pbaAccounts are being submitted to /organisations endpoint on the Api successfully, and this can be seen through a GET request to /organisations.
- dxAddress and contacts are NOT being submitted as yet; a 500 is currently being returned from the endpoint. JIRA tickets PUID-103 and PUID-104 have been raised so that the endpoint can be fixed around this. When this is done we can complete this task.
- Unit tests for code written.

### CODE REVIEW

File: api/services/rd-professional.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import * as log4js from 'log4js'
+import config from '../lib/config'
+import { http } from '../lib/http'
+
+const logger = log4js.getLogger('rd-professional')
```
Comment (smogese) [inline api/services/rd-professional.ts]: This needs to be changed to log4jui and the level line can be removed

File: api/tslint.json [hunk 2]
```diff
@@ -6,6 +6,8 @@
     "jsRules": {},
     "rules": {
         "object-literal-key-quotes": false,
+        "object-literal-sort-keys": false,
```
Comment (smogese) [inline api/tslint.json]: Sort keys is generally a good idea, why is it disabled?

Comment (ficklephil) [reply api/tslint.json]: When keys are sorted, you're not able to keep keys that have a relation to each other close to each other. For example firstName and lastName.

**Comment by ficklephil** [comment]
> So far looking pretty good. A few queries here.

Thanks Alan.

File: api/services/rd-professional.ts (hunk 1 above)
Comment (ficklephil) [reply api/services/rd-professional.ts]: Logging removed as not required.

File: api/tslint.json (hunk 2 above)
Comment (ficklephil) [reply api/tslint.json]: Reverted this change, so that tslint checks for the sort order of keys again.

File: api/tslint.json [hunk 3]
```diff
@@ -6,6 +6,7 @@
     "jsRules": {},
     "rules": {
         "object-literal-key-quotes": false,
+        "no-trailing-whitespace": false,
```
Comment (smogese) [inline api/tslint.json]: no trailing white space  false?

**Pull request merged**